examples/JavaCC/src/main/javacc/JavaCC.jj

```diff
@@ -653,19 +657,22 @@ void node_descriptor_expression()
     if (tok.kind == 0) {
       throw new ParseException();
     }
-    if (tok.kind == LPAREN) nesting++;
-    if (tok.kind == RPAREN) {
+    if (tok.kind == LPAREN) {
+      nesting++;
+    } else if (tok.kind == RPAREN) {
       nesting--;
-      if (nesting == 0) break;
+      if (nesting == 0) {
+        break;
+      }
     }
     tok = getNextToken();
   }
 }
 
 
-/**********************************************
- * THE JAVA GRAMMAR SPECIFICATION STARTS HERE *
- **********************************************/
+/*--------------------------------------------
+ * THE JAVA GRAMMAR SPECIFICATION STARTS HERE
+ *--------------------------------------------*/
```

Comment:
In javacc-8 we want to remove the java grammar completely out of the javacc grammar if possible.

Reply:
that sounds like agood improvement too, but in scope of this PR I only wanted to improve the code style of generated classes without changing their functionality.